### DISCUSSION TITLE
Pycharm lint problem

### DISCUSSION THREAD

**Discussion by changja88** [pydantic/pydantic#11047] [Question] [answered]
`_original_state: dict[Any, Any] = PrivateAttr(default_factory=dict)`

Pycham says "Mutable default PrivateAttr(default_factory=dict) is not allowed. Use default factory"
How can I fix it?
(I am using python 3.13.0, pydantic 2.10.3)

Thanks :)

**Comment by Viicos** [comment, marked as answer]
This is likely strongly related to this PyCharm bug: https://youtrack.jetbrains.com/issue/PY-77433/.
